### ISSUE TITLE
TEM data not segmenting well in version 3.0

### ISSUE THREAD

**Issue by mathieuboudreau** [neuropoly/axondeepseg#293]
Just discovered this while looking into #249 (but I think this problem is unrelated to that one).

* Downloaded the most recent version of AxonDeepSeg
* Downloaded one of the TEM images used in for the training on OSF as a sample dataset  to test with: https://osf.io/uewd9/
* Ran the command line segmentation call: axondeepseg -t TEM -i image.png -s 0.00236 
* Output was very poorly segmented (see below. Images in order of the gif are: Raw image, manually segmented mask, ADS mask).

![out](https://user-images.githubusercontent.com/1421029/79281832-2e920a00-7e8e-11ea-9333-24d656513f9b.gif)

I tried the same steps with SEM, but that segmentation resulted ok:

![out](https://user-images.githubusercontent.com/1421029/79281846-3782db80-7e8e-11ea-9e4c-fff7d4320495.gif)

**Comment by mathieuboudreau** [comment]
I'm currently re-processing the TEM model, I'll inform you folks of if this resolved the issue when it's done.

**Comment by panpeterlin** [comment]
Hi,  I also downloaded the latest version and tried it on test TEM image and it did not work well. Have you solved this issue or not?

**Comment by mathieuboudreau** [comment]
I tried training a new model and it did not work, which was suspicious since training worked well for SEM data. The only main difference between the two in the code that I could think of was in fact the block of code talked about in #249, so I removed the TEM condition and it ended up segmenting well with the old trained model. So all that should be needed is merging PR #294 - no need to re-upload/download a trained TEM model.

@panpeterlin we'll let you know when it's resolved in the `master` branch, currently the fix is only in the branch `tem_flip`. You'll have to pull master again after it's merged.
